### ISSUE TITLE
Panorama/360° Mode support 

### ISSUE THREAD

**Issue by F3000** [SimpleMobileTools/Simple-Camera#118]

**Comment by Klimmbimmel** [comment]
Yes, that would be nice. Than the android default camera app could be deactivated without missing anything, and that would mean a usability gain (never choosing "with which app you want to open this action").
